### ISSUE TITLE
Help with Regex

### ISSUE THREAD

**Issue by sasha-volodin** [destruc7i0n/shulker#67]
Hi! Help me pls.
Earlier all working:
latest.log:
[21:39:48] [Async Chat Thread - #65/INFO]: **<nickname> msg**
Regex: "REGEX_MATCH_CHAT_MC": "\\[Async Chat Thread - #\\d*\/INFO\\]: <([^>]*)> (.*)"

But, i change chat cfg:
latest.log:
[21:36:20] [Async Chat Thread - #64/INFO]: **nickname: msg**
And now msg mc to dc not send. This regex not correct. Help me with regex pls for new chat cfg

**Comment by sasha-volodin** [comment]
Anybody ?

**Comment by destruc7i0n** [comment]
Can you send a line from the log enclosed with backticks (``` on both sides) to fix formatting?

**Comment by sasha-volodin** [comment]
log from mine or ?

**Comment by destruc7i0n** [comment]
Yeah your log line. In the GitHub editor, when pasting, first click on "Insert Code" in the comment box and then paste the line from your log.

**Comment by sasha-volodin** [comment]
`[22:06:21] [Async Chat Thread - #2/INFO]: sunlight: some msg`

**Comment by destruc7i0n** [comment]
Try this: `^([^:]*): (.*)`

**Comment by sasha-volodin** [comment]
not working (

**Comment by destruc7i0n** [comment]
Oh I see you're still using `REGEX_MATCH_CHAT_MC`. I'd recommend updating to the latest version.
For `REGEX_MATCH_CHAT_MC`, the regex would be `\[Async Chat Thread - #\d*\/INFO\]: ([^:]*): (.*)`

If this doesn't work as well, can you post your full config.json enclosed with the "`" character?

**Comment by sasha-volodin** [comment]
Doesn't work. Config:
`    "PORT": 8000,

    "USE_WEBHOOKS": true,
    "WEBHOOK_URL": "",
    "DISCORD_TOKEN": "",
    "DISCORD_CHANNEL_ID": "",
    "DISCORD_MESSAGE_TEMPLATE": "`%username%`: %message%",

    "MINECRAFT_SERVER_RCON_IP": "127.0.0.1",
    "MINECRAFT_SERVER_RCON_PORT": 25575,
    "MINECRAFT_SERVER_RCON_PASSWORD": "",
    "MINECRAFT_TELLRAW_TEMPLATE": "[{\"color\": \"white\", \"text\": \"<%username%> %message%\"}]",

    "IS_LOCAL_FILE": true,
    "LOCAL_FILE_PATH": "/home/server/logs/latest.log",

    "ALLOW_USER_MENTIONS": false,

    "WEBHOOK": "/minecraft/hook",
    "REGEX_MATCH_CHAT_MC": "\\[Async Chat Thread - #\\d*\/INFO\\]: <([^>]*)> (.*)",
     "REGEX_IGNORED_CHAT": "packets too frequently",
    "RCON_RECONNECT_DELAY": 10,
    "DEBUG": false`

**Comment by destruc7i0n** [comment]
```
"REGEX_MATCH_CHAT_MC": "\\[Async Chat Thread - #\\d*\/INFO\\]: ([^:]*): (.*)",
```
This line should work based on the string you sent from my local testing, are you getting any other errors?
